components/brave_ads/browser/BUILD.gn

```diff
@@ -119,6 +121,12 @@ static_library("browser") {
       "//ui/views",
     ]
   }
+
+  if (brave_adaptive_captcha_enabled) {
+    sources += [ "ads_tooltips_delegate.h" ]
+    deps +=
+        [ "//brave/components/brave_adaptive_captcha:brave_adaptive_captcha" ]
```

Comment:
this file does not need a dep on brave_adaptive_captcha

Reply:
Removed.